artifacthub-repo.yml

```diff
@@ -1,5 +1,5 @@
 ---
-repositoryID: <THE REPOSITORY ID INSIDE OF ARTIFACTHUB> # obtained after publishing the policy for the 1st time
+repositoryID: 8008dd14-3c2c-4328-b31b-aed07b65a2b3
 owners:
 - name: José Guilherme Vanz
```

Comment:
shouldn't we use kubewarden email account here? That is what we do for other policies https://github.com/kubewarden/allow-privilege-escalation-psp-policy/blob/main/artifacthub-repo.yml and https://github.com/kubewarden/volumeMounts-policy/blob/main/artifacthub-repo.yml

Reply:
Done